packages/synapse-react-client/src/lib/containers/DialogBase.tsx

```diff
@@ -0,0 +1,82 @@
+import CloseIcon from '@mui/icons-material/Close'
+import {
+  Box,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogProps,
+  DialogTitle,
+  IconButton,
+  Stack,
+  SxProps,
+} from '@mui/material'
+import React from 'react'
+import { HelpPopover } from './HelpPopover'
+
+export type CloseButtonProps = {
+  sx?: SxProps
+  onClick?: () => void
+}
+
+export const CloseButton: React.FC<CloseButtonProps> = ({
+  sx = { color: 'grey.700' },
+  onClick,
+}) => {
+  return (
+    <IconButton sx={sx} onClick={onClick} aria-label="close">
+      <CloseIcon />
+    </IconButton>
+  )
+}
+
+export type DialogBaseProps = {
+  open: boolean
+  title: string | JSX.Element
+  content: string | JSX.Element
+  actions?: JSX.Element
```

Comment:
React.ReactNode is anything that React can render, so it includes string and JSX Elements
```suggestion
  title: React.ReactNode
  content: React.ReactNode
  actions?: React.ReactNode
```

Reply:
Oh great! Done in the next commit